axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/EventBuffer.java

```diff
@@ -57,14 +60,14 @@
  */
 public class EventBuffer implements TrackingEventStream {
 
-    private static final Logger logger = LoggerFactory.getLogger(EventBuffer.class);
+    private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-    private static final int DEFAULT_POLLING_TIME_MILLIS = 500;
+    private static final int MIN_AWAIT_AVAILABLE_DATA = 500;
```

Comment:
This should be MAX_AWAIT_AVAILABLE_DATA, is this is the maximum amount of time that a thread sleeps in between attempts

Reply:
Good point, will adjust.